src/renderer/wire.ts

```diff
@@ -0,0 +1,20 @@
+const { rpc } = require('electron-simple-rpc');
+import { extensionScope, Channel, Api } from '../common';
+import { IExtension } from '../common/types';
+
+export const createWire = <T>(api: Api, extensionId: IExtension['id']) => {
```

Comment:
What is the purpose of wire? You had in the ts-doc

Reply:
I will add it in the TS DOC